storm-core/src/clj/org/apache/storm/config.clj

```diff
@@ -41,11 +41,30 @@
   (dofor [f (seq (.getFields Config))]
          (.get f nil)))
 
-
+;; TODO this function and its callings will be replace when nimbus and supervisor move to Java
```

Comment:
Don't need the TODO.  Please file a separate JIRA to remove the code when it is no longer needed.  You can leave the comment, but preferably with the JIRA number instead.